packages/frontend/electron/src/main/deep-link.ts

```diff
@@ -61,51 +61,28 @@ async function handleAffineUrl(url: string) {
   logger.info('open affine url', url);
   const urlObj = new URL(url);
   logger.info('handle affine schema action', urlObj.hostname);
-  // handle more actions here
-  // hostname is the action name
-  if (urlObj.hostname === 'signin-redirect') {
-    await handleOauthJwt(url);
-  }
+
   if (urlObj.hostname === 'bring-to-front') {
     const mainWindow = await getMainWindow();
     if (mainWindow) {
       mainWindow.show();
     }
+  } else {
+    await openUrl(urlObj);
   }
 }
 
-async function handleOauthJwt(url: string) {
```

Comment:
what if user uses old client but login with the new server's auth impl?

Reply:
the '/desktop-signin' route is kept in frontend and old client's request send it as `redirect_uri`. there should be any problem. I will test it anyway